src/include/optimizer/statistics/top_k_elements.h

```diff
@@ -52,31 +53,20 @@ class TopKElements {
 
   /**
    * Increase the count for the given key by the specified delta.
-   * This is a convenience method for those KeyTypes that have the
-   * correct size defined by the sizeof method
    * @param key the key to target
    * @param delta the amount to increase the key's count
    */
-  void Increment(const KeyType &key, const uint32_t delta) { Increment(key, sizeof(key), delta); }
-
-  /**
-   * Increase the count for the given key by the specified delta.
-   * @param key the key to target
-   * @param key_size the length of the key's data
-   * @param delta the amount to increase the key's count
-   */
-  void Increment(const KeyType &key, const size_t key_size, const uint32_t delta) {
+  void Increment(const KeyType &key, const uint32_t delta) {
     NOISEPAGE_ASSERT(delta >= 0, "Invalid delta");
 
     // Increment the count for this item in the sketch
-    sketch_->Increment(key, key_size, delta);
+    sketch_->Increment(key, delta);
 
     // If this key already exists in our top-k list, then
     // we need to update its entry
     auto entry = entries_.find(key);
     if (entry != entries_.end()) {
       entries_[key] += delta;
-      OPTIMIZER_LOG_TRACE("Increment Key[{0}] => {1} // [size={2}]", key, entries_[key], GetSize());
```

Comment:
Is there a reason this log trace was removed? Similar with others.

Reply:
They didn't compile with all the different key types, and I just thought it wasn't worth the effort to get it to work since they don't add a ton of value. 

I'd be happy to try and add them back though if you disagree.

Reply:
I think tracing is something we generally moved away from after Peloton. Instead we now generally use debuggers, unless you have something multithreaded/over the network going on.

Otherwise if you enabled this, optimizer log trace is really not quite the granularity you want anyway -- if you're trying to trace the actual optimizer you're going to get a ton of spam.